src/Modules/Admin/Commands.Admin/help/Add-PowerBIEncryptionKey.md

```diff
@@ -0,0 +1,116 @@
+---
+external help file: Microsoft.PowerBI.Commands.Admin.dll-Help.xml
+Module Name: MicrosoftPowerBIMgmt.Admin
+online version:
+schema: 2.0.0
+---
+
+# Add-PowerBIEncryptionKey
+
+## SYNOPSIS
+Adds an encryption key
+
+## SYNTAX
+
+### Default (Default)
+```
+Add-PowerBIEncryptionKey -Name <String> -KeyVaultKeyUri <String> [-Default] [<CommonParameters>]
+```
+
+### DefaultAndActivate
+```
+Add-PowerBIEncryptionKey -Name <String> -KeyVaultKeyUri <String> [-Default] [-Activate] [<CommonParameters>]
+```
+
+### Activate
+```
+Add-PowerBIEncryptionKey -Name <String> -KeyVaultKeyUri <String> [-Activate] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The commandlet will add an encryption key.
```

Comment:
Remove cmdlet name

Reply:
Look at examples of other doc for reference, for example https://docs.microsoft.com/en-us/powershell/module/microsoftpowerbimgmt.reports/Get-PowerBIReport?view=powerbi-ps

Reply:
State what permissions is needed to invoke